### ISSUE TITLE
Chrome -> Firefox: Stream not starting if video access denied

### ISSUE THREAD

**Issue by dnish** [muaz-khan/RTCMultiConnection#197]
Hey @muaz-khan,
just checked today different kind of connections. The following one isn't working with current Firefox and Chrome:

Chrome -> Firefox:

Chrome user allows access to camera and mic - Firefox user only allows access to mic, but not to his camera. In this case, both clients receive a stream object. The Chrome user get's an audio stream which we bind on an <code><audio></code> element. It's working fine - but the Firefox user only sees a black screen without any video from the Chrome user.

If we do the same with opposite direction ( Firefox -> Chrome), everything is working fine on both sides.

**Comment by muaz-khan** [comment]
Please test following demo among Chrome (stable) and Firefox (nightly):

* https://rtcmulticonnection.herokuapp.com/demos/Video-Conferencing.html

Make sure that **you have TWO UNIQUE cameras**.

Both Firefox & Chrome MUST be using UNIQUE_cameras.

I just tested above demo in both directions; and it worked. I'm using "Logitech HDCam pro" as secondary/external camera.

**Comment by dnish** [comment]
Ah okay, good to know. I've tested it with Chrome 51 (MacOS) and Firefox 45 (MacOS). I did the test again with Chrome 51 (MacOS) and Firefox 45 (Android 6) and still get the same behavior. Only Firefox -> Chrome is working.

On both sides, <code>onStream</code> is fired for the remote stream, but on Firefox, the video stream of the Chrome user doesn't start playing, while the audio stream of the Firefox user is playing fine.

**Comment by dnish** [comment]
@muaz-khan Okay, just checked it again with <code>userStream.getVideoTracks()</code>. 

It seems like that on Chrome (Video, Mic) -> Firefox (Mic), Firefox doesn't get any video stream of the Chrome user - but the audio stream.

**Comment by muaz-khan** [comment]
Just tested between Nexus6 Firefox 47 and MacOSX Chrome 51.0.2704.103. 

Tried both directions; and it worked as well.

Please check Firefox app's settings/permissions; and make sure that camera is enabled/allowed.

On my nexus6, I enabled everything for Firefox; e.g. Camera, Contacts, Location, Microphone, Storage.

**Comment by dnish** [comment]
@muaz-khan Yeah, if both users allow camera access, everything is working fine. But in my case, the Firefox user doesn't allow camera access but microphone access. And there I've got the problem that Chrome -> Firefox only sends audio streams on both users, while Firefox -> Chrome is sending audio and video streams (in this case only the video stream of the Chrome user).

Only the Chrome user granted access to microphone and camera.
